tests/cypress/integration/dashboard/dashboard.cy.js

```diff
@@ -0,0 +1,137 @@
+			'Products and Services',
+			'Pages',
+			'Additional Features',
+			'Launch Your Store/Site Status',
+		] );
+	} );
+
+	it( 'see Store Info, Payments, Shipping and Tax info cards in General Setting tab', () => {
+		GeneralSetting.verifyGeneralSettingTabHaveCard( [
+			'Store Info',
+			'Payments',
+			'Shipping',
+			'Tax Info',
+		] );
+		GeneralSetting.checkLinkTextIsPresentOnGeneralSettingCardsOnHover();
+	} );
+
+	it( 'add my Store address from "General Setting" "Store Info" card', () => {
+		GeneralSetting.addStoreAddress();
+		GeneralSetting.verifyStoreInfoCardIsInDoneState();
+		GeneralSetting.verifyEnteredStoreAddressIsSameOnWooCommerceSetting();
+	} );
+
+	it( 'see the option to link my existing payment method in General Setting', () => {
+		GeneralSetting.linkExistingPaymentAccount();
+	} );
+
+	it( 'link my existing shippo account in General Setting', () => {
+		GeneralSetting.linkExistingShippoAccount();
+		GeneralSetting.verifyShippingCardIsInDoneState();
+		GeneralSetting.verifyFilledShippingInformationSavedInYith();
+	} );
+
+	it( 'configure "Yes, enable tax rates and calculations" from "General setting" "Tax Info" card', () => {
+		GeneralSetting.selectTaxOption( 0 );
+		GeneralSetting.verifyTaxInfoCardInDoneState();
+		GeneralSetting.verifyTaxOptionIsEnableInWooCommerce();
+		GeneralSetting.setStandardRate();
+	} );
+
+	it( 'see "Add a product" and "Import Product" cards in Your Product tab', () => {
+		AddProducts.verifyCardsExists( [ 'Add Products', 'Import Products' ] );
+	} );
+
+	it( 'add a product from "Your product" tabs', () => {
+		AddProducts.addAProduct();
+		cy.deleteAllProducts();
+		cy.visit( homePageUrl );
+		AddProducts.ImportProducts();
+	} );
+
+	it( 'Verify Add a Product, Manage a Product, Categories, Tags cards is visible after adding a product', () => {
+		AddProducts.verifyCardsExists( [
+			'Add a product',
+			'Manage products',
+			'Categories',
+			'Tags',
+		] );
+	} );
+
+	it( "verify 'Add a Product', 'Manage a Product', 'Categories', 'Tags' redirecting to right page after click", () => {
+		AddProducts.verifyAddaProductCardNavigatedToRightPage();
+		AddProducts.verifyManageaProductCardNavigatedToRightPage();
+		AddProducts.verifyCategoriesCardNavigatedToRightPage();
+		AddProducts.verifyTagsCardNavigatedToRightPage();
+	} );
+
+	it( "verify 'How to add product' card exist", () => {
+		AddProducts.verifyHowToAddProductCardExist();
+	} );
+
+	it( "verify 'customize Your Store' cards exist", () => {
+		CustomizeYourStore.verifyCard( [
+			'Home Page',
+			'About Page',
+			'Contact Page',
+			'Store Layout',
+			'Customer Account Page',
+		] );
+	} );
+
+	it( 'create home page, about page, contcat page', () => {
+		CustomizeYourStore.createPages();
+	} );
+
+	it( 'change my store layout', () => {
+		Cypress.on( 'uncaught:exception', () => {
+			// returning false here to prevents Cypress from failing the test
+			// Application Error: Cannot read properties of null (reading 'addEventListener')
```

Comment:
This seems like a bug in the application that should probably be fixed. Is this an error originating from our code or a third-party plugin?